lib/lib.js

```diff
@@ -6,15 +6,25 @@ const co = require('co');
  * Make a promise outside of a yieldable.
  * To avoid unhandled callbacks resulting in unacked messages piling up in amqp,
  * the returned promise will fail after a specified timeout.
- * @param {Promise|Generator|Function} yieldable a yieldable (generator or promise)
+ * @param {Promise|GeneratorFunction} yieldable a yieldable (generatorFunction or promise)
```

Comment:
`yieldable` actually is a Promise or a generator or a function (cb style or generator function), i.e. what co accepts.
=> calling `yieldable()` here won't work for all those cases, only `GeneratorFunction`s